lib/matplotlib/backends/backend_wxcairo.py

```diff
@@ -6,8 +6,8 @@
 import wx
 
 from .backend_cairo import cairo, FigureCanvasCairo, RendererCairo
-from .backend_wx import (
-    _BackendWx, _FigureCanvasWxBase, FigureFrameWx, NavigationToolbar2Wx)
+from .backend_wx import _BackendWx, _FigureCanvasWxBase, FigureFrameWx
+from .backend_wx import NavigationToolbar2Wx as NavigationToolbar2WxCairo
```

Comment:
can go in the previous line (would prefer this) as
```
from .backend_wx import (
    _BackendWx, _FigureCanvasWxBase, FigureFrameWx,
    NavigationToolbar2Wx as NavigationToolbar2WxCairo)
```

Reply:
Done, also for wxagg.